{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "b61cd14a_c0c54e6d",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 7
      },
      "lineNbr": 0,
      "author": {
        "id": 1017622
      },
      "writtenOn": "2023-02-03T14:57:08Z",
      "side": 1,
      "message": "recheck undercloud-restore",
      "revId": "f62a6bfced472f95b6c3bdf2fb09504f5f6abc66",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735"
    },
    {
      "unresolved": false,
      "key": {
        "uuid": "4e7e8b77_a3e199ff",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 7
      },
      "lineNbr": 0,
      "author": {
        "id": 1010634
      },
      "writtenOn": "2023-02-09T21:44:32Z",
      "side": 1,
      "message": "LGTM but please have a look at the comment in patchset4",
      "revId": "f62a6bfced472f95b6c3bdf2fb09504f5f6abc66",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735"
    }
  ]
}